Source/include/Ably/ARTRealtimeChannelOptions.h

```diff
@@ -42,6 +42,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) ARTChannelMode modes;
 
+/**
+ * Determines whether calling `subscribe` on a channel or presence object should trigger an implicit attach. Defaults to `true`.
```

Comment:
This needs to be in ably/sdk-api-reference. Also, shouldn't the documentation for `attach` be updated too to explain the impact of this option?

Reply:
https://github.com/ably/ably-cocoa/pull/1976/commits/1625e1b4567972f42d9b5887ff30cb9802ddc572

Reply:
also https://github.com/ably/sdk-api-reference/pull/42

Reply:
Updated docstrings both here and in the api reference.